README.md

```diff
@@ -24,15 +24,28 @@ Enumerates program capabilities and malicious behaviors using fragment analysis.
 
 ## Requirements
 
-* go 1.21+
-* yara 4.3+ library - you can use this one-liner to install it if necessary:
+A container runtime environment such as Podman or Docker, or local developer tools:
+
+* [go](https://go.dev/) 1.21+
+* [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) - included in many UNIX distributions
+* [yara](https://virustotal.github.io/yara/) - you can use this one-liner to install it:
```

Comment:
The one liner comment is now in a wrong place?

And maybe rejigger this to now live in the 'Local' section below? Since they are not needed for the containerized version?

Reply:
good spot - PTAL.